### PR TITLE
Make loader agnostic to ingest name

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: tests/testHtmIndex.py [hunk 1]
```diff
@@ -239,6 +242,16 @@ def testIngest(self):
         cent = make_coord(*tupl)
         cat = loader.loadSkyCircle(cent, self.search_radius, filterName='a')
         self.assertTrue(len(cat) > 0)
+        # test that a catalog can be loaded even with a name not used for ingestion
+        butler = dafPersist.Butler(self.test_repo_path)
+        config = LoadIndexedReferenceObjectsConfig()
+        config.ref_dataset_name = "diff_ref_name"
```
Comment (PaulPrice) [inline tests/testHtmIndex.py]: This is a magic string used in multiple places.  Can you consolidate them into a single variable?

File: tests/testHtmIndex.py [hunk 2]
```diff
@@ -239,6 +242,16 @@ def testIngest(self):
...
+        loader = LoadIndexedReferenceObjectsTask(butler=butler, config=config)
+        # This location is known to have objects
+        tupl = (93.0, -90.0)
+        cent = make_coord(*tupl)
```
Comment (PaulPrice) [inline tests/testHtmIndex.py]: Is there a reason not to strip out the `tupl` variable and just to `make_coord(93.0, -90.0)`?

File: tests/testHtmIndex.py (hunk 1 above)
Comment (SimonKrughoff) [inline tests/testHtmIndex.py]: Good point.  Done.

File: tests/testHtmIndex.py (hunk 2 above)
Comment (SimonKrughoff) [inline tests/testHtmIndex.py]: Another good point.  I also consolidated so I don't define `cent` twice.

Comment (timj) [inline tests/testHtmIndex.py]: Saves me from moaning that `tupl` is a terrible name for a variable 😉

**Pull request merged**